rubicon-htb.js

```diff
@@ -134,7 +136,7 @@ function RubiconModule(configs) {
                 continue;
             }
             if (__sizeToSizeIdMapping[sizeKey] === Number(rubiconSize)) {
-                return Size.stringToArray(sizeKey)[0];
+                return Size.stringToArray(sizeKey);
```

Comment:
Can you please explain the reason behind this change?

Reply:
I don't have access to your Utility Libraries so not sure what stringToArray function is returning. I have coded one in libraryStubData. Could you please share the Utility Libraries? Thanks,

Reply:
Assuming  the string is `300x250,300x600` the Size.stringToArray(str) will return `[[300,250], [300,600]]`.

Unfortunately we can't share the Utility Library with you right now, but we definitely appreciate this feedback and will think of ways to provide you with more information regarding the utility/other helper libraries.